react-time-track/src/components/list-projects.js

```diff
@@ -24,6 +25,8 @@ function ListProjects() {
       .catch(response => navigate("/login"));
   }, []);
 
+  // console.log("Projects:", projects);
+
```

Comment:
Hi, what do you think about removing those comments?

Reply:
Yes my lich